four.js

```diff
@@ -0,0 +1,8 @@
+var wrapper = document.getElementById("wrapper")
+
+wrapper.onload = function myFunction() {
+  var p = document.createElement("p");
+  var t = document.createTextNode("I am the superior text");
+  p.appendChild(t);
+  document.body.appendChild(p);
```

Comment:
you need to grab a div with `getElementById` and then swap out the text node. But you're going in the right direction.